pkgs/development/tools/rust/cargo-llvm-cov/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, rustPlatform, fetchCrate, fetchFromGitHub, libllvm, makeBinaryWrapper }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-llvm-cov";
+  version = "0.4.14";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-DY5eBSx/PSmKaG7I6scDEbyZQ5hknA/pfl0KjTNqZlo=";
+  };
+
+  postPatch = ''
+    rm tests/test.rs
+  '';
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+  buildInputs = [ libllvm ];
+
+  LLVM_COV = "${rustPlatform.rust.rustc.llvmPackages.llvm}/bin/llvm-cov";
+  LLVM_PROFDATA = "${rustPlatform.rust.rustc.llvmPackages.llvm}/bin/llvm-profdata";
+
+  postInstall = ''
+    wrapProgram $out/bin/cargo-llvm-cov \
+      --set LLVM_COV "${rustPlatform.rust.rustc.llvmPackages.llvm}/bin/llvm-cov" \
+      --set LLVM_PROFDATA "${rustPlatform.rust.rustc.llvmPackages.llvm}/bin/llvm-profdata"
```

Comment:
I don't think wrapping this is a good idea, since it prevents users from choosing their own version of the Rust toolchain. For example, if I'm writing a flake that uses `fenix`, `cargo-llvm-cov` wouldn't be able to see the version of `llvm-tools-preview` from it, which might cause problems due to version disparities.

Reply:
Good point.

I wonder if just leaving it unwrapped is good UX, though -- will users know to set those environment variables intuitively?

Reply:
The command prints out a pretty helpful error message in situations where the environment isn't set up right:

```
$ cargo llvm-cov
error: failed to find llvm-tools-preview, please install llvm-tools-preview, or set LLVM_COV and LLVM_PROFDATA environment variables
```